TTHAnalysis/cfg/run_susyMultilepton_cfg.py

```diff
@@ -44,8 +44,14 @@
 # Lepton Skimming
 ttHLepSkim.minLeptons = 2
 ttHLepSkim.maxLeptons = 999
-#ttHLepSkim.idCut  = ""
-#ttHLepSkim.ptCuts = []
+ttHLepSkim.allowLepTauComb = True
```

Comment:
To be removed (should be false for ttH and SOS, already set to true inside an if later)